### PR TITLE
Fixes #13655 - Add in the i18n tooling this the hammer-cli-foreman-ta…

### PR DESCRIPTION
**Pull request by bkearney**
…sks plugin

### CODE REVIEW

File: hammer_cli_foreman_tasks.gemspec [hunk 1]
```diff
@@ -17,8 +17,9 @@ Gem::Specification.new do |s|
 Contains the code for showing of the tasks (results and progress) in the Hammer CLI.
 DESC
 
-  s.files = Dir["{lib,config}/**/*", "LICENSE", "README.md"]
+  s.files = Dir["{lib,config,locale}/**/*", "LICENSE", "README.md"]
+  s.require_paths = ["lib"]
 
   s.add_dependency "powerbar", "~> 1.0.11"
-  s.add_dependency "hammer_cli_foreman", "> 0.1.1", "< 1.0.0"
+  s.add_dependency "hammer_cli_foreman", "> 0.1.1"
```
Comment (stbenjam) [inline hammer_cli_foreman_tasks.gemspec]: Why remove the upper bound?

File: locale/en/hammer_cli_foreman_tasks.po [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: English\n"
+"Language: en\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"\n"
+
+msgid "Id"
```
Comment (stbenjam) [inline locale/en/hammer_cli_foreman_tasks.po]: I think we standardized on `ID` everywhere else, if you wouldn't mind fixing this :grin:

File: hammer_cli_foreman_tasks.gemspec (hunk 1 above)
Comment (bkearney) [inline hammer_cli_foreman_tasks.gemspec]: This was me debugging.. sorry.. reverted this.

**Comment by bkearney** [comment]
@stbenjam reverted the change, and changed all Id to ID

**Comment by stbenjam** [comment]
Looks good to me, thanks. Do you need a new gem released with the translations in them?

**Comment by bkearney** [comment]
@stbenjam yes please, if you could do a new release it would help me out.

**Comment by stbenjam** [comment]
Sure, done https://github.com/theforeman/foreman-packaging/pull/1029

**Pull request merged**